src/components/Aug4thWeek.scss

```
@@ -0,0 +1,122 @@
+.container {
+  overflow: hidden;
+  background-color: rgba(255, 255, 255, 1);
+  width: 393px;
+  height: 852px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center; 
+  align-items: center; 
+  transform: translate(-50%, -50%);
+
+  .header {
+    width: 100%;
+    height: 66px;
+    background-color: rgba(242, 242, 247, 1);
+    top: 0;
+    left: 0;
+    right: 0;
+
+    .logo {
+      width: 36px;
+      height: 36px;
+      background-color: rgba(199, 199, 204, 1);
+      border-radius: 50%;
+      position: absolute;
+      top: 15px;
+      left: 20px;
+    }
+
+    .company-name {
+      color: rgba(28, 28, 30, 1);
+      font-family: Pretendard;
+      font-size: 26px;
+      position: absolute;
+      top: 14px;
+      left: 20px;
+
+      .profile-icon {
+        width: 35px;
+        height: 35px;
+        margin-left: 230px;
+      }
+    }
+
+    .separator {
+      background-color: rgba(199, 199, 204, 1);
+      width: 393px;
+      height: 1px;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+    }
+  }
+
+  .content {
+    width: 393px;
+    height: 786px;
+    position: absolute;
+    top: 90px;
+    left: 0;
+
+    .item {
+      width: 393px;
+      height: 112px;
+      position: absolute;
+      left: 0;
+
+      .item-info {
+        width: 151px;
+        height: 84px;
+        position: absolute;
+        left: 20px;
+        top: -12px;
+
+        .item-name {
+          color: rgba(28, 28, 30, 1);
+          font-family: Pretendard;
+          font-size: 22px;
+        }
+
+        .pronunciation {
+          color: rgba(142, 142, 147, 1);
+          font-family: Pretendard;
+          font-size: 17px;
+          margin-left: 5px;
+        }
+
+        .item-type {
+          color: rgba(142, 142, 147, 1);
+          font-family: Pretendard;
+          font-size: 17px;
+          margin-top: 24px;
+
+          .bookmark-icon {
+            width: 30px;
+            height: 30px;
+            margin-left: 100px;
+            margin-top: -70px;
+          }
+        }
+
+        .item-example {
+          color: rgba(0, 122, 255, 1);
+          font-family: Pretendard;
```

font-family, font-size, font-style, font-weight, line-height 를 직접 지정하시는 것보다 TextStyle에 있는 body01-medium를 불러오는 것을 권장드립니다.
```suggestion
          @include body01-medium;
```